libs/Format/Confluence/ContentTypes/Markdown/ContentType.php

```diff
@@ -69,7 +69,7 @@ function daux_loadCSS(url) {
 daux_ready(function() {
     var mermaidBlocks = document.querySelectorAll("pre.mermaid");
     if (mermaidBlocks.length) {
-        daux_loadJS(`https://cdn.jsdelivr.net/npm/mermaid@8.9.1/dist/mermaid.min.js`, function() {
+        daux_loadJS(`https://cdn.jsdelivr.net/npm/mermaid@8.13.3/dist/mermaid.min.js`, function() {
```

Comment:
Can you please also update `daux_libraries/mermaid.min.js`
I can merge and release right after